systemtest/src/test/java/io/strimzi/systemtest/kafka/TieredStorageST.java

```diff
@@ -201,6 +203,18 @@ void setup() throws IOException {
 
         ImageBuild.buildImage(suiteStorage.getNamespaceName(), IMAGE_NAME, TIERED_STORAGE_DOCKERFILE, BUILT_IMAGE_TAG, Environment.KAFKA_TIERED_STORAGE_BASE_IMAGE);
 
+        if (cluster.isKind()) {
+            final String image = Environment.getImageOutputRegistry(
+                suiteStorage.getNamespaceName(), IMAGE_NAME, BUILT_IMAGE_TAG);
+            // Kaniko pushes the image to the local registry, but Podman stores it in an internal storage
+            // that is not directly accessible to Kind. Podman must first pull the image from the registry
+            // so that it is available in the Podman daemon's local storage before "kind load" can work.
+            if (ContainerRuntimeUtils.getRuntime().equals("podman")) {
+                Exec.exec("sudo", ContainerRuntimeUtils.getRuntime(), "pull", image);
+            }
+            // if container-runtime is docker we do not need to pull
+            Exec.exec("sudo", "kind", "load", "docker-image", image, "--name", "kind-cluster");
```

Comment:
I guess we will have to think about a different way how to load it, or take the name of the Kind cluster first. Because not all of the Kind clusters are named `kind-cluster`.

Reply:
wouldn't be easier just skip this test with podman runtime + kind ?

Reply:
I had discussion with @Frawless 
I agree that having another kind of conditions around that just for one test doesn't make sense.

For AZP, it will run without issues. For other like OCP as well. So let's skip the test for kind + podman @see-quick

Reply:
Thanks, I will update this accordingly.